lib/src/package.dart

```diff
@@ -108,6 +108,9 @@ class Package {
     }));
   }
 
+  /// Returns whether or not this package is under source control system.
+  bool get isUnderSourceControl => [_inGitRepo].any((e) => e);
```

Comment:
As it is a public property, I thought it will make more sense to name it more general. E.g. Mercurial support will be added.

Reply:
I think it's fine to just make `_inGitRepo` public here. We can always change it later if we do indeed add support for other systems.